tests/functional/analyze_and_parse/test_analyze_and_parse.py

```diff
@@ -107,6 +107,12 @@ def __check_one_file(self, path, mode):
                 # replace timestamps
                 line = re.sub(r'\[\w+ \d{4}-\d{2}-\d{2} \d{2}:\d{2}\]',
                               '[]', line)
+
+                # Replace full path only to file name.
```

Comment:
os.sep can be used to get the os dependent separator.

Please provide an example line what should be matched by this regex.